### ISSUE TITLE
training termination

### ISSUE THREAD

**Issue by bhralz** [shekkizh/FCN.tensorflow#42]
Dear all,
I have two question about the training process ?
1- what is the termination criteria for learning? it seems to work very long time, and maybe not need in some problem.
2-How I can stop training manually and restore last saved net(structure and weights) to use in testing the network?
in another word if I stop training manually(I dot know how?), can I use the saved net in testing?

**Comment by shekkizh** [comment]
1. Use a validation set and stop when you have an acceptable loss. This is not a problem specific to this model, a little research on machine models and training might be helpful.

2. The code saves model regularly which can be restored.
